src/kite.js

```diff
@@ -121,9 +121,17 @@ const Kite = {
         new KiteCompletionProvider(Kite, completionsTriggers, optionalCompletionsTriggers), ...completionsTriggers.concat(optionalCompletionsTriggers))
     );
 
+    // Side effect: will suppress native completions due to https://github.com/microsoft/vscode/issues/11935#issuecomment-246778493
+    const alphanumeric = [
+      'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+      'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
+      'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
+      'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
+      '1', '2', '3', '4', '5', '6', '7', '8', '9', '0'
```

Comment:
Let's leave out numbers for now since we don't suggest number literals. We can add this back in when lexical models can suggest number constants.